core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestUtils.java

```diff
@@ -236,10 +247,18 @@ public static S3Exception toBucketS3Exception(Exception exception, String resour
    *
    * @param exception Exception thrown when process s3 object rest request
    * @param resource object complete path
+   * @param auditContext the audit context for exception
    * @return instance of {@link S3Exception}
    */
-  public static S3Exception toObjectS3Exception(Exception exception, String resource) {
+  public static S3Exception toObjectS3Exception(Exception exception, String resource,
+                                                @Nullable AuditContext auditContext) {
```

Comment:
Is this supposed to be `alluxio.proxy.s3.S3AuditContext`? I don't have a preference, just want them to be consistent.